.github/workflows/redirects_page.yml

```diff
@@ -0,0 +1,37 @@
+name: Generate and Deploy Redirects
+
+on:
+  workflow_dispatch:
```

Comment:
I think it would be reasonable to have a cron job that runs once every 6 hours (in addition to the `workflow_dispatch`).